test/unittest/dds/participant/ParticipantTests.cpp

```diff
@@ -17,6 +17,7 @@
 #include <future>
 #include <memory>
 #include <sstream>
+#include <stdlib.h>
```

Comment:
Re-order after the change please
```suggestion
#include <cstdlib>
```